Integrations/EDL/EDL.py

```diff
@@ -113,7 +145,8 @@ def create_values_out_dict(iocs: list) -> dict:
     return {EDL_VALUES_KEY: list_to_str(formatted_indicators, '\n')}
 
 
-def get_edl_ioc_values(on_demand, limit, indicator_query='', last_run=None, cache_refresh_rate=None) -> str:
+def get_edl_ioc_values(on_demand, limit, indicator_query='', last_run=None, cache_refresh_rate=None,
+                       panos_compatible: bool = True, url_ps: bool = False) -> str:
```

Comment:
lets change `url_ps` to `url_port_stripping`

Reply:
Only here, or every function?

Reply:
every

Reply:
... makes sense.